src/scss/components/_scenarios.scss

```diff
@@ -1,6 +1,10 @@
+* {
+  margin: 0 auto;
+  max-width: 450px;
+}
 .scenarios {
-  padding-bottom: 230px;
-  padding-top: 30px;
+  padding: 30px 30px 30px 230px;
```

Comment:
The right value is `padding: 30px 30px 230px;`, otherwise you would be applying a `230px` padding to the left instead of bottom.

Reply:
Done!